styles/style.css

```diff
@@ -0,0 +1,12 @@
+header{
+    font-family: 'Roboto', sans-serif;
+     color:#c4c4c4;
+    font-size: 200px;
+    color: C4C4C4;
```

Comment:
You can remove this line, this color doesn't exists and you are setting this a few lines above.
```suggestion
```